topics/fair/tutorials/fair-gtn/tutorial.md

```diff
@@ -0,0 +1,98 @@
+
+## Improve findability of your training materials by properly describing them
+
+Describing your training materials with structured metadata is fundamental to making them FAIR resource. Creating metadata boost findability and preserve information and can be updated after publication. GTN  tutorial requier several mandatory metadata information such as learning objectives following Bloom’s taxonomy {% cite chevron2014metacognitive %}, prerequisites, time estimate, or questions addressed by the tutorial. Content developers needs to know where to find all of the available metadata to reference it later. [Schema.org](https://schema.org/) is a collaborative project with a mission to create the addition of structured metadata to web pages. It describes data using shared vocabulary. It is in this spirit that the [BioSchemas](https://bioschemas.org/) community initiative was created to extend the Schema.org standard to life-science resources with training content specification. GTN offers schemas command line [tutorial](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) where metadata can be added or updated. This reduces the complexity of keywords and set the scene for controlled vocabularies enviroment. This increases the effectiveness of metadata filtering, which decreases ambiguity and makes it easier to find and retrieve information.
+
+## Give your training materials a unique identity
+
+To support teachers and trainers, the GNT tutorials rely on specific tags and identifiers. Adding persistent identifiers (PIDs) to training materials makes them easy to cite and aids citation counting in research metric systems {% cite mcmurry2017identifiers %}. Providing PIDs ensure that tag/identifier will continue to work even if webpage location changes. Data used for tutorials are requierd to be stored at [Zenodo](https://zenodo.org/) and assosciated with [Digital Object Identifiers](https://www.doi.org/) (DOI). 
+
+## Register your training materials online
+Sharing and publishing with the GTN training helps minimize the amount of time and effort required for instructors to prepare for and run their training courses and workshops, by providing templates and a complete training infrastructure. Online registry makes your content more discoverable and accessible to wider community. GTN tutorials are automatically registered on [TeSS](https://tess.elixir-europe.org/) (ELIXIR Training e-support system).
```

Comment:
it could be interesting, somewhere in this file, to reference https://training.galaxyproject.org/training-material/faqs/gtn/fair_training.html